packages/vuetify/src/util/globals.ts

```diff
@@ -0,0 +1,24 @@
+function checkForEventListenerPassiveSupport (supported = false): boolean {
```

Comment:
`supportsPassiveEvents` or `isPassiveSupported` or similar for more meaningful name (`check*` doesn't clearly describes if the function returns anything, maybe it throws, maybe just checks and does nothing else)

What is the `supported` arg for? It's not used atm

Reply:
I like `supportsPassiveEvents`. As far as supported, it most certainly is used. It returns on line 11.

Reply:
But argument is never passed in the call

Reply:
oh i see
https://github.com/vuetifyjs/vuetify/pull/10313/files#diff-458958ea84a6f6fa44737a696fa436b5R4